bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/designer/core/repository/WebPageRepositoryStore.java

```diff
@@ -53,7 +53,7 @@ public String getDisplayName() {
 
     @Override
     public Image getIcon() {
-        return Pics.getImage("page.png", PageDesignerPlugin.getDefault());
+        return Pics.getImage("page.png", UIDesignerPlugin.getDefault());
```

Comment:
I'm not a fan of using PicsConstants.
The constant should be decalred the closest possible place of where it used.
PicsConstants is relevant for images reused everywhere. This is not the case.